tests/backup/backup_kdmp_test.go

```diff
@@ -0,0 +1,721 @@
+							fileConfig = PodDirectoryConfig{
+								BasePath:          mountPath,
+								FilesPerDirectory: 1,
+								FileName:          ".snapshot",
+							}
+							err = CreateFilesInPodDirectory(pod, fileConfig)
+							dash.VerifyFatal(err, nil, fmt.Sprintf("creating files within mountPath [%s] with hidden type and add to exclude list", mountPath))
+							storageClassExcludeFileDirMap[scName] = append(storageClassExcludeFileDirMap[scName], fileConfig.FileName)
+							mountPathExcludeFileDirMap[mountPath] = append(mountPathExcludeFileDirMap[mountPath], fileConfig.FileName)
+							log.Infof(fmt.Sprintf("creating file within mountPath [%s] with valid special chars ", mountPath))
+							fileConfig = PodDirectoryConfig{
+								BasePath:          mountPath,
+								FilesPerDirectory: 1,
+								FileName:          "myn@meisunkn*wn",
+							}
+							err = CreateFilesInPodDirectory(pod, fileConfig)
+							dash.VerifyFatal(err, nil, fmt.Sprintf("creating files within mountPath [%s] valid special chars and add to exclude list", mountPath))
+							storageClassExcludeFileDirMap[scName] = append(storageClassExcludeFileDirMap[scName], fileConfig.FileName)
+							mountPathExcludeFileDirMap[mountPath] = append(mountPathExcludeFileDirMap[mountPath], fileConfig.FileName)
+
+							log.Infof(fmt.Sprintf("creating file within mountPath [%s] with maximum name length (255 characters)", mountPath))
+							fileConfig = PodDirectoryConfig{
+								BasePath:          mountPath,
+								FilesPerDirectory: 1,
+								FileName:          fmt.Sprintf("%s.txt", RandomString(251)),
+							}
+							err = CreateFilesInPodDirectory(pod, fileConfig)
+							dash.VerifyFatal(err, nil, fmt.Sprintf("creating files within mountPath [%s] with maximum name length (255 characters and add to exclude list)", mountPath))
+							storageClassExcludeFileDirMap[scName] = append(storageClassExcludeFileDirMap[scName], fileConfig.FileName)
+							mountPathExcludeFileDirMap[mountPath] = append(mountPathExcludeFileDirMap[mountPath], fileConfig.FileName)
+
+						}
+					}
+				}
+			}
+		})
+		Step("Update KDMP config map on source cluster by formatting storage class name and random files and directories as a string", func() {
+			log.InfoD("Update KDMP config map on source cluster by formatting storage class name and random files and directories as a string")
+			for _, namespace := range bkpNamespaces {
+				pods, err := core.Instance().GetPods(namespace, nil)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("getting pods from namespace [%s] ", namespace))
+				for _, pod := range pods.Items {
+					if len(podMountPathMap[pod.Name]) > 0 {
+						for _, mountPath := range podMountPathMap[pod.Name] {
+							excludeFileDirList := make([]string, 0)
+							log.Infof(fmt.Sprintf("Fetch some random directories from created list %v", dirListMountMap[mountPath]))
+							randomDirs, err := GetRandomSubset(dirListMountMap[mountPath], 2)
```

Comment:
Please make it 100 atleast for this release.

Reply:
Made this to 500 directories and 500 files. Now the configMap update will show the size of CM as well before and  after updation.

02:14:04  2023-12-07 20:44:01 +0000:[INFO] [{ExcludeDirectoryFileBackup}] [backup.UpdateKDMPConfigMap:#5588] - The intial size of configMap [kdmp-config] : [1043] and final size [79199]